### ISSUE TITLE
What to do with "Please let me know about..."

### ISSUE THREAD

**Issue by bethlakshmi** [bethlakshmi/gbe-divio-djangocms-python2.7#22]
https://burlesque-expo.us.aldryn.io/update_profile

Includes a "please let me know about..." section on the right hand side.

I'm working on #17, of letting people tell us what they *don't* want email about...

And I'm thinking about the thought that we SHOULD contact them about what they DO want know about.  ...Or else remove this as useless information....  We have the following options:

-  Exhibiting Art or Costumes
-  Performing
-  Pre-event Organizing
-  Sponsoring/Advertising
-  Teaching
-  Vending
-  Volunteering at the Expo

My proposal - in the "contact bidders by email" screen, offer an additional option for "people interested in..." that includes all these options.  These are roughly analagous to the bid types, so it makes some sense that you may want something like "all class proposers AND people who say they are interested in teaching" for example.

After implementing #17, I may have a proposal for how to handle the conflict - for example, what happens if the user says they don't want any email based on having previously bid... but they DO want to hear about future teaching opportunities?

**Comment by burlexpo** [comment]
I like the direction this is heading.

**Comment by bethlakshmi** [comment]
@info - this is what we were recently talking about.
Will do this blended in with bidding, as that's where we are usually soliciting future participation via email.  That will let us set up bidding & interested-in profiles when making "come do this thing" emails in a way that keeps us from hitting the same user twice.

**Comment by bethlakshmi** [comment]
first pass at what the bulk email select looks like:

<img width="1792" alt="Screenshot 2023-11-13 at 9 46 30 PM" src="https://github.com/bethlakshmi/gbe-divio-djangocms-python2.7/assets/4542707/8fd35925-570f-47d9-aafc-7fef72052fda">

I make "Profile Interest" not required since you may want to explicitly select only actual bidders.

However, because Bidder & Status & Conference IS required, you have to select something.

Some of these interests do not coincide with bidding - for example, there is no "bid" thing for Sponsoring/Advertising.  


Proposal:
- remove the requirement that there be a set of filters that applies to Bids.
- The user MUST select - either a bid type OR a profile interest --  if they don't they get yelled at.

@burlexpo  - thoughts?  Does that make sense?

**Comment by burlexpo** [comment]
Yes, that makes sense.  Also, is it possible to add a little space between the different groups? It's now really hard for me to tell which check boxes are part of which categories.

**Comment by bethlakshmi** [comment]
<img width="1792" alt="Screenshot 2023-11-15 at 4 19 31 PM" src="https://github.com/bethlakshmi/gbe-divio-djangocms-python2.7/assets/4542707/d40c7760-4186-4357-b513-114e884f7e18">

Done!

I think I got the logic right - that if you do any checkbox related to filtering a Bid, you have to provide a choice for all three (error messages tell you this).  But you can now select an interest and NOT a bid.

This also affects the email opt-out flow - since if you are getting this mail because you said you were interested, you should turn off the interest, not the "don't email me because I bid on something" switch.  So I'm now updating that flow to include interest selections as something you can turn off to stop emails.

**Comment by bethlakshmi** [comment]
Closed!  Now deployed on the production site!!
I'm very pleased to be closing our second oldest bug!  
Also very pleased we can actually do something with all this data we've been collecting
